bag_verifier/src/main/scala/uk/ac/wellcome/platform/archive/bagverifier/fixity/azure/AzureFixityChecker.scala

```diff
@@ -16,7 +15,7 @@ class AzureFixityChecker(implicit blobClient: BlobServiceClient)
   override protected val sizeFinder =
     new AzureSizeFinder()
 
-  override val tags = new AzureBlobMetadata()
+  override val tags = None
```

Comment:
We should have a comment here that explains why we don't tag objects in Azure, possibly linking to the ticket and the Azure documentation.